### PR TITLE
Adjusts grinder behaviour and towercap log contents

### PR DESCRIPTION
**Pull request by Migratingcocofruit**
## What Does This PR Do

Adds the chems obtained by grinding down the planks of towercap and steelcap logs to the logs themselves. Also reduces the amount of carbon per log/plank of towercap to a more sensible 20u per log, and 4 per plank. Additionally, inedible plants are now ground in the same manner as edible plants.

## Why It's Good For The Game

Consistency good.

## Images of changes

cottonlog1
cottonlog2
cottonlog3

## Testing

ground some logs and cotton, observed as the expected chemicals were extracted, made disks from the new chems.

## Changelog
:cl:
tweak: Towercap logs now contain 20% Carbon(20u at 100 potency).
tweak: Planks now grind into 4 Carbon, down from 20.
tweak: Steelcap logs now contain 50% Iron(50u at 100 potency).
tweak: Inedible plants are now ground in the same manner as edible plants.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/machinery/reagentgrinder.dm [hunk 1]
```diff
@@ -462,6 +459,40 @@
...
+						O.reagents.remove_reagent("plantmatter", min(O.reagents.get_reagent_amount("plantmatter"), space))
+				else
+					if(O.reagents.has_reagent("nutriment"))
+						beaker.reagents.add_reagent(r_id, min(round(O.reagents.get_reagent_amount("nutriment") * abs(amount) * efficiency), space))
+						O.reagents.remove_reagent("nutriment", min(O.reagents.get_reagent_amount("nutriment"), space))
+					if(O.reagents.has_reagent("plantmatter"))
+						beaker.reagents.add_reagent(r_id, min(round(O.reagents.get_reagent_amount("plantmatter") * abs(amount) * efficiency), space))
+						O.reagents.remove_reagent("plantmatter", min(O.reagents.get_reagent_amount("plantmatter"), space))
+			else
+				O.reagents.trans_id_to(beaker, r_id, min(amount, space))
```
Comment (DGamerL) [inline code/modules/reagents/chemistry/machinery/reagentgrinder.dm:489]:
```suggestion
			if(amount == 0)
				if(O.reagents.has_reagent("nutriment"))
					beaker.reagents.add_reagent(r_id, min(O.reagents.get_reagent_amount("nutriment") * efficiency, space))
					O.reagents.remove_reagent("nutriment", min(O.reagents.get_reagent_amount("nutriment"), space))
				if(O.reagents.has_reagent("plantmatter"))
					beaker.reagents.add_reagent(r_id, min(O.reagents.get_reagent_amount("plantmatter") * efficiency, space))
					O.reagents.remove_reagent("plantmatter", min(O.reagents.get_reagent_amount("plantmatter"), space))
			else if(amount < 0)
					if(O.reagents.has_reagent("nutriment"))
						beaker.reagents.add_reagent(r_id, min(round(O.reagents.get_reagent_amount("nutriment") * abs(amount) * efficiency), space))
						O.reagents.remove_reagent("nutriment", min(O.reagents.get_reagent_amount("nutriment"), space))
					if(O.reagents.has_reagent("plantmatter"))
						beaker.reagents.add_reagent(r_id, min(round(O.reagents.get_reagent_amount("plantmatter") * abs(amount) * efficiency), space))
						O.reagents.remove_reagent("plantmatter", min(O.reagents.get_reagent_amount("plantmatter"), space))
			else
				O.reagents.trans_id_to(beaker, r_id, min(amount, space))
```

Comment (Migratingcocofruit) [reply code/modules/reagents/chemistry/machinery/reagentgrinder.dm:489]: so just removing this bit at the beginning? 
>if(amount == 0)

Comment (Migratingcocofruit) [reply code/modules/reagents/chemistry/machinery/reagentgrinder.dm:489]: oh nevermind I understand

Comment (Migratingcocofruit) [reply code/modules/reagents/chemistry/machinery/reagentgrinder.dm:489]: Needs to be done for the edible plants instead

Comment (Migratingcocofruit) [reply code/modules/reagents/chemistry/machinery/reagentgrinder.dm]: done

**Review by DGamerL — APPROVED** [review]
This part of the grinder should get a massive refactor/rework, but that's heavily out of scope for this PR

Code LGTM

**Review by Burzah — APPROVED** [review]

**Pull request merged**